### PR TITLE
Added python 3.9 and 3.10 to CI

### PR DESCRIPTION
**Pull request by euri10**
Working fine locally !

### CODE REVIEW

**Comment by florimondmanca** [comment]
Thanks :-)

**Comment by florimondmanca** [comment]
Should we bump trove classifiers too?

**Comment by euri10** [comment]
> Should we bump trove classifiers too?

done @florimondmanca !

File: ci/azure-pipelines.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ stages:
             py37:
             py38:
               coverage: true
+            py39:
+            py310:
```
Comment (florimondmanca) [inline ci/azure-pipelines.yml:37]: You need to update the templates version to `refs/tags/4.0` I think. I only added py39 and py310 support recently :) Maybe you'll need to add `pythonVersion` parameter to the `publish` step as well, see here: https://github.com/florimondmanca/azure-pipelines-templates/tree/master#job--python-publishyml

**Review by florimondmanca — APPROVED** [review]
Yay :)

**Pull request merged**